daemon/src/wallet.rs

```diff
@@ -607,4 +618,26 @@ mod tests {
             .unwrap()
             .expect("single UTXO to be available after unlocking it");
     }
+
+    #[tokio::test]
+    async fn given_insufficient_funds_then_error_when_building_party_params() {
+        let mut tasks = Tasks::default();
+
+        let actor = Actor::new_offline(Amount::from_btc(0.1).unwrap(), 1, Duration::from_secs(120))
+            .unwrap()
+            .create(None)
+            .spawn(&mut tasks);
```

Comment:
🔧 This is a bit more convenient via our custom `TokioGlobalSpawnExt` trait.

```suggestion
            .spawn_global();
```